opentelemetry-exporters-newrelic-auto/src/main/java/com/newrelic/telemetry/opentelemetry/export/auto/NewRelicSpanExporterFactory.java

```diff
@@ -10,13 +10,13 @@
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.net.URI;
 
+/**
+ * A {@link SpanExporterFactory} that creates a {@link SpanExporter} that sends spans to New Relic.
+ */
 public class NewRelicSpanExporterFactory implements SpanExporterFactory {
 
-  private static final String NEW_RELIC_API_KEY = "newrelic.api.key";
-  private static final String NEW_RELIC_ENABLE_AUDIT_LOGGING = "newrelic.enable.audit.logging";
-  private static final String NEW_RELIC_SERVICE_NAME = "newrelic.service.name";
-  private static final String NEW_RELIC_URI_OVERRIDE = "newrelic.uri.override";
-  private static final String DEFAULT_NEW_RELIC_SERVICE_NAME = "(unknown service)";
+  // this should not be used, now that we have both span and metric exporters.
+  static final String NEW_RELIC_URI_OVERRIDE = "newrelic.uri.override";
```

Comment:
Is it a problem that there isn't symmetry between `newrelic.uri.override` and `newrelic.metric.uri.override`?

Reply:
Mainly, that this was the original property, before we added metrics, and it only applies to spans. so, this is the "old" property that now should not be being used. Maybe we could just delete it, since we never actually documented its usage?